Makefile

```diff
@@ -127,69 +100,187 @@ config: ## Set up default config
 	@mkdir -p $(HOME)/.aurae/pki
 	@cp -v pki/* $(HOME)/.aurae/pki
 
-tlsinfo: ## Show TLS Info for /var/run/aurae*
-	./hack/server-tls-info
+.PHONY: musl
+musl: ## Add target for musl
+	rustup target add $(uname_m)-unknown-linux-musl
 
-.PHONY: pki
-pki: certs ## Alias for certs
-certs: clean-certs ## Generate x509 mTLS certs in /pki directory
-	mkdir -p pki
-	./hack/certgen
-	sudo -E mkdir -p /etc/aurae/pki
-	sudo -E cp -v pki/* /etc/aurae/pki
-	@echo "Install PKI Auth Material [/etc/aurae]"
+#------------------------------------------------------------------------------#
 
+# Clean Commands
+
+.PHONY: clean-crates
+clean-crates: ## Clean target directory
+	@cargo clean
+
+.PHONY: clean-certs
 clean-certs: ## Clean the cert material
 	@rm -rvf pki/*
 
-fmt: headers ## Format the entire code base(s)
-	@./hack/code-format
+.PHONY: clean-gen
+clean-gens: ## Clean gen directories
+	@rm -rf aurae-proto/src/gen/*
+	@rm -rf auraescript/gen/*
+
+#------------------------------------------------------------------------------#
+
+# Protobuf Commands
 
 .PHONY: proto
-proto: proto-lint ## Generate code from protobuf schemas
+proto: proto-lint ## Lint and Generate code from protobuf schemas
 	@buf --version >/dev/null 2>&1 || (echo "Warning: buf is not installed! Please install the 'buf' command line tool: https://docs.buf.build/installation"; exit 1)
 	buf generate -v api
 
-.PHONY: cri
-cri: ## Copy the CRI interface from upstream
-	curl https://raw.githubusercontent.com/kubernetes/cri-api/$(cri_version)/pkg/apis/runtime/v1/api.proto -o api/kubernetes/cri/v1/$(cri_version).proto
-
 .PHONY: proto-lint
 proto-lint: ## Lint protobuf schemas
 	buf lint api
 
-.PHONY: clean
-clean: clean-certs
-	@rm -rf aurae-proto/src/gen/*
-	@cargo clean
+.PHONY: proto-vendor-cri
+proto-vendor-cri: ## Copy the CRI interface from upstream
```

Comment:
should this be `proto-vendor` and include the grpc proto?

Reply:
Maybe. I'll look into vendoring the grpc health proto